Documentation/docs-mobile/building-apps/build-properties.md

```diff
@@ -1019,6 +1019,25 @@ specifying the Java package names of generated Java source code.
 The only supported value is
 `LowercaseCrc64`.
 
+## AndroidPrepareForBuildDependsOn
+
+A semi-colon delimited property that can be used to extend the
+Android build process. MSBuild targets added to this property
+will execute early in the build for both Application and Library
+project types. This property is empty by default.
+
+Example:
+
+```xml
+<PropertyGroup>
+  <AndroidPrepareForBuildDependsOn>MyPrepareTarget;</AndroidPrepareForBuildDependsOn>
```

Comment:
Example value shouldn't include `;` by default, unless you want to show *multiple* targets in-use.

Reply:
Updated